apps/docs/scripts/wrap-snippets.ts

```diff
@@ -116,6 +121,14 @@ export const wrapSnippet = (filepath: string) => {
 */
 const dir = 'src/**';
 const src = `${dir}/*.ts`;
-const ignore = [`src/typegend/**`, `src/env.ts`, `src/transactions/new-api.ts`, `${dir}/*.test.ts`];
+const ignore = [
+  `src/typegend/**`,
+  `src/env.ts`,
+  `src/vite.config.ts`,
+  `src/versions.data.ts`,
+  `src/jsonc.d.ts`,
+  `src/guide/transactions/new-api.ts`,
```

Comment:
Not introduced on this PR but does this file exist? Not seeing it

Reply:
f5f024c1fbca47fdec05d41c6461bf2c0e7d1b70